js/exportsms.js

```diff
@@ -355,13 +374,34 @@ var exportsms = function() {
           choice = configObject.choices[j];
           if (endLevelGroupSuccessFailureStatus == choice.flag.toUpperCase() && !choice.next) {
             choice.next = parseInt(passageDatum.optinpath, 10);
-            partialStory[nameString] = configObject;
             break;
           }
         }
+
+        nextLevelNumber = parseInt(levelNumber, 10) + 1
+        if (nextLevelNumber <= TOTAL_NUMBER_OF_LEVELS) {
+          var nextLevelString = 'L' + nextLevelNumber + '0'
+          for (var storyNode in config.story) {
```

Comment:
`storyNode` is already declared at the top of this function. Don't need the `var` here.